javaparser-core/src/main/java/com/github/javaparser/ParseProblemException.java

```diff
@@ -0,0 +1,24 @@
+package com.github.javaparser;
+
+import java.util.List;
+
+/**
+ * Thrown when parsing problems occur during parsing with the static methods on JavaParser.
+ */
+public class ParseProblemException extends Exception {
```

Comment:
Me when I code I prefere to create RuntimeException, Or using an another Pattern because with exception we can't do composition.If you want to talk about, PM me on gitter :smile: otherwise it's fine :smile_cat: